profile_library/shelly/Shelly Pro 3EM 3CT63/eth+bt/model.json

```diff
@@ -0,0 +1,6 @@
+{
+  "measure_description": "The device measured with Ethernet port and BlueTooth enabled. Other interfaces were disabled",
```

Comment:
You might add `name` properties now to the Sub profiles for cleaner select options.
They will be shown now in new version I will release shortly.

Reply:
No problem.

Just a thought. Currently it mostly works like subprofile attributes simply overrides values from main model.json.
Setting name in the subprofile we are loosing information set in the parent.

I suppose it doesn't break anything. But I would expect name used for device name matching (in fact directiry name is used, isn't it?)

It feels to me inconsistent a bit. But it is how it is, I assume.

Reply:
`name` isn't used anywhere in the integration code, besides the sub profile listing now.
It doesn't have any place in matching logic, so it's also safe to override that value from sub profile.
Only purpose for name is in the library: https://library.powercalc.nl

Reply:
I just pushed a commit with added profile names